src/builtins/ulimit.cpp

```diff
@@ -200,10 +220,22 @@ maybe_t<int> builtin_ulimit(parser_t &parser, io_streams_t &streams, const wchar
                 what = RLIMIT_DATA;
                 break;
             }
+#ifdef RLIMIT_NICE
+            case 'e': {
+                what = RLIMIT_NICE;
+                break;
+            }
+#endif
             case 'f': {
                 what = RLIMIT_FSIZE;
                 break;
             }
+#ifdef RLIMIT_SIGPENDING
+            case 'i': {
```

Comment:
I'm assuming these option letters are just taken from bash?

Reply:
Most of them aren't in bash, so I've taken them from [prlimit(1)](https://man7.org/linux/man-pages/man1/prlimit.1.html).